src/app/app.component.ts

```diff
@@ -45,6 +91,9 @@ export class AppComponent implements OnInit {
       this.host[1] = event.lang;
       this.initializeSocialShareSeoService();
     });
+    if (this.isBrowser()) {
```

Comment:
Use `isPlatformBrowser` to check if current agent is browser, use PLATFORM_ID

Reply:
done